### ISSUE TITLE
Build and publish wheels for python using maturin

### ISSUE THREAD

**Issue by tripcher** [sbdchd/squawk#261]
Hey! I think that will be great if we can use squawk in python. For example, for integration with alembic. I would like to install squawk from pip `pip install squawk` and use it. 
it seems we can use [maturin](https://github.com/PyO3/maturin) for building and publishing.

I can try to add maturin if you don't mind.

**Comment by sbdchd** [comment]
yeah I'm down for adding it, I used npm because it was easiest to distribute binaries (all the npm does is run a install script which downloads the binaries from GitHub)
